lib/models/hooks.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = HooksModelFactory;
+
+HooksModelFactory.$provide = 'Models.Hooks';
+HooksModelFactory.$inject = [
+    'Model',
+    'Services.Configuration',
+    'uuid'
+];
+
+function HooksModelFactory (Model, configuration, uuid) {
+
+    return Model.extend({
+        connection: configuration.get('databaseType', 'mongo'),
+        identity: 'hooks',
+        attributes: {
+            id: {
+                type: 'string',
+                uuidv4: true,
+                primaryKey: true,
+                unique: true,
+                required: true,
+                defaultsTo: function() { return uuid.v4(); }
+            },
+            name: {
+                type: 'string',
+                required: false
+            },
+            url: {
+                type: 'string',
+                required: true
+            },
+            filters: {
+                type: 'array',
```

Comment:
Yes, `filters` is an array of a list of rules, if any rule is satisfied, then the event will be pushed into the hook via the hook url.

This is something like user may have interest on multiple topics (node, poller and graph), set the `filters` as `array` will give user the ability to subscribe multiple topics.